app.js

```diff
@@ -18,12 +19,14 @@ let countries = [];
 /* Events */
 window.addEventListener('load', async () => {
     // > Part A: call findCountries (with no arguments)
-
+    findCountries();
     // > Part B: await the call to get continents to get the response
-
+    const response = await getContinents();
     // > Part B: Assign to state the:
     //      - error,
     //      - data (to the continents variable)
+    error = response.error;
+    continents = response.data;
```

Comment:
get used to taking a response and doing something with its error and something with its data